planner/core/cache.go

```diff
@@ -174,16 +174,16 @@ func (s FieldSlice) Equal(tps []*types.FieldType) bool {
 	return true
 }
 
-// PSTMTPlanCacheValue stores the cached Statement and StmtNode.
-type PSTMTPlanCacheValue struct {
+// PlanCacheValue stores the cached Statement and StmtNode.
```

Comment:
Please add more comments for this structure as we discussed yesterday. We should explain our guiding principle here about what fields should be put in PlanCacheValue instead of PlanCacheKey.